### PR TITLE
Add position filter if is blade to localization column

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/data_center/admin.py [hunk 1]
```diff
@@ -219,25 +220,47 @@ def get_multiadd_fields(self, obj=None):
...
+            position = generate_html_url(
+                base_url,
+                {
+                    'position__start': obj.position,
+                    'position__end': obj.position
+                },
+                '<strong>{}</strong>'.format(obj.position)
+            )
+        else:
+            position = '<strong>{}</strong>'.format(obj.position or '')
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: you could put this before this if (and then use it as a label)

File: src/ralph/data_center/admin.py [hunk 2]
```diff
@@ -219,25 +220,47 @@ def get_multiadd_fields(self, obj=None):
...
     def localization(self, obj):
+        """
+        Additional column 'localization' display filter by:
+        data center, server_room, rack, position (if is blade)
+        """
         base_url = reverse('admin:data_center_datacenterasset_changelist')
+        if obj.is_blade:
+            position = generate_html_url(
+                base_url,
+                {
```
Comment (mkurek) [inline src/ralph/data_center/admin.py:232]: this filter will not work :)

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**